### PR TITLE
Modificari in test Andreea

### PR DESCRIPTION
**Pull request by drgmircea**

### CODE REVIEW

File: src/test/java/pageObjects/LogInPage.java [hunk 1]
```diff
@@ -44,4 +45,24 @@ public void aleggustari() {
     public void cartofi() {
         cartofiselect.click();
     }
+
+     */
+    @FindBy(xpath = "//div[@class='right']//a[@class='link'][normalize-space()='Cont']//*[name()='svg']//*[name()='g' and @id='user-_1_']//*[name()='path' and @id='Path_35617']")
```
Comment (drgmircea) [inline src/test/java/pageObjects/LogInPage.java]: Use     @FindBy(xpath = "//div[@class='right']//a[@class='link'][normalize-space()='Cont']")

File: src/test/java/pageObjects/LogInPage.java [hunk 2]
```diff
@@ -44,4 +45,24 @@ public void aleggustari() {
...
+    public void personalinformation (){ personalInformationSelect.click();
+
+    }
+
+    @FindBy(xpath = "//input[@class='input-with-value input-focused']")
+    WebElement numeSelect;
+    public void modificareNume (String nume){
+        numeSelect.click();
+        numeSelect.sendKeys(nume);
+    }
```
Comment (drgmircea) [inline src/test/java/pageObjects/LogInPage.java:68]: Please add 

    @FindBy(xpath = "//a[contains(text(),'Salvează modificările')]")
    WebElement salveazaNume;
    public void salvare (){
        salveazaNume.click();
    }
    @FindBy(xpath = "(//span[@class='input-clear-button'])[1]")
    WebElement clearName;
    public void stergenume (){
        clearName.click();
    }

File: src/test/java/testCases/Cont/TC_001_ModifyPersonalInformation.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            logger.info("Clicked on Login button");
+
+            logger.info("Going to login page");
+            LogInPage regpage = new LogInPage(driver);
+
+            regpage.setEmail("andreea.oananastac8@gmail.com");
+            regpage.password("Chavi27an");
+
+            regpage.clickLogin();
+            //Thread.sleep(2000);
```
Comment (drgmircea) [inline src/test/java/testCases/Cont/TC_001_ModifyPersonalInformation.java:25]: uncomment this thread.sleep

File: src/test/java/testCases/Cont/TC_001_ModifyPersonalInformation.java [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+            regpage.contInformation();
+            Thread.sleep(2000);
```
Comment (drgmircea) [inline src/test/java/testCases/Cont/TC_001_ModifyPersonalInformation.java:28]: comment this thread sleep

File: src/test/java/testCases/Cont/TC_001_ModifyPersonalInformation.java [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+            regpage.personalinformation();
```
Comment (drgmircea) [inline src/test/java/testCases/Cont/TC_001_ModifyPersonalInformation.java:30]: add this line after personal information 
regpage.stergenume();

File: src/test/java/testCases/Cont/TC_001_ModifyPersonalInformation.java [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+            regpage.modificareNume("Oana");
+
```
Comment (drgmircea) [inline src/test/java/testCases/Cont/TC_001_ModifyPersonalInformation.java:32]: add lines after modificarenume 
            logger.info("Nume scris");
            regpage.salvare();
            logger.info("Nume salvat");

**Review by drgmircea — COMMENTED** [review]
Te rog sa te uiti la review-ul lasat. Trebuie sa mearga acum

File: src/test/java/pageObjects/LogInPage.java (hunk 2 above)
Comment (Andreeann88) [reply src/test/java/pageObjects/LogInPage.java:68]: Sa sterg ce am scris eu si sa pun ce ai adaugat tu ?

**Comment by Andreeann88** [comment]
I added the changes sent by you and my test is running. It shows me every change. Thank you

File: src/test/java/testCases/Cont/TC_001_ModifyPersonalInformation.java (hunk 6 above)
Comment (Andreeann88) [reply src/test/java/testCases/Cont/TC_001_ModifyPersonalInformation.java:32]: Here I put "Oana" where you write "Nume scris" and "Nume salvat". It s ok like this?

Comment (drgmircea) [reply src/test/java/testCases/Cont/TC_001_ModifyPersonalInformation.java:32]: "Nume salvat " and "Nume scris" are not in the function modificarenume. There are for info logger.
After the function modificarenume("Oana") you will put the next function to press the saving button.

**Review by drgmircea — COMMENTED** [review]
put more verification in the tests made with assert and if

**Pull request closed without merging**